inst/sql/sql_server/sqlOnly/cte_sql_for_results_table.sql

```diff
@@ -0,0 +1,35 @@
+/*********
+SQL to create query for insertion into results table. These may be unioned together prior to insert.
+Note that this does not include information about SQL errors or performance.
+**********/
+
+SELECT 
+  cte.num_violated_rows
+  ,cte.pct_violated_rows
+  ,cte.num_denominator_rows
+  ,'' as execution_time
+  ,'Query @query_num' as query_text
+  ,'@check_name' as check_name
+  ,'@check_level' as check_level
+  ,'@check_description' as check_description
+  ,'@cdm_table_name' as cdm_table_name
+  ,'@cdm_field_name' as cdm_field_name
+  ,'@concept_id' as concept_id
+  ,'@unit_concept_id' as unit_concept_id
+  ,'@sql_file' as sql_file
+  ,'@category' as category
+  ,'@subcategory' as subcategory
+  ,'@context' as context
+  ,'' as warning
+  ,'' as error
+  ,'@checkid' as checkid
+  ,0 as is_error
+  ,0 as not_applicable
```

Comment:
i don't _love_ that this approach doesn't use not_applicable status (but understand why). this makes its output inconsistent with what we'd get from DQD. what do you think about adding another function that the user can run to recalculate the not_applicable status?

i guess in this SQL only world that might be another SQL query that updates rows in the results table?

Reply:
@katy-sadowski , not_applicable was implemented after I build the initial PR last last March.  Where should I look to understand how it is supposed to work?  Is it a matter of simply filtering out queries/checks that are NA?

Reply:
@katy-sadowski , is it fair to say that whenever num_denominator_rows = 0 then not_applicable should = 1?  If so, that is an easy change.  For sqlOnly, that should address all cases where there are missing concept or concept+unit combinations.  It would not address situations where a table or field are missing (for in those cases,  the query itself would fail).

Reply:
I modified this assuming that if num_denominator_rows = 0 then not_applicable = 1.  If that doesn't behave as expected, I can revert this.

Reply:
While this is a good proxy, the regular NA status is a bit more complex and uses the result of other checks.
To avoid confusion, we should either exactly replicate the NA status or leave it out completely.

Reply:
OK - reverted to 0 for NA

Reply:
👍  yeah I was thinking an additional R function (or SQL query) that looks at the whole results table and then evaluates NA status. on board with @MaximMoinat 's suggestion below to tackle that in a future issue/PR.